packages/dashboard/src/customization/widgets/lineSymbol/plugin.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import LineWidgetComponent from './component';
+import LineSymbolIcon from './icon';
+import type { DashboardPlugin } from '~/customization/api';
+import { LineWidget } from '../types';
+
+export const linePlugin: DashboardPlugin = {
+  install: ({ registerWidget }) => {
+    registerWidget<LineWidget>('line', {
+      render: (widget) => <LineWidgetComponent {...widget} />,
+      componentLibrary: {
+        name: 'Line Symbol',
+        icon: LineSymbolIcon,
+      },
+      properties: () => ({
+        lineStyle: 'solid',
+        color: 'black',
```

Comment:
I think we should only define the properties that are required here so that we are not specifying defaults multiple times.

Reply:
cool, keeping only the points then.